pytorch/training/docker/1.13/py3/Dockerfile.cpu

```diff
@@ -1,6 +1,6 @@
 ARG PYTHON=python3
```

Comment:
Lets do a diff in pacakges between the prod image and this image to see if there are any missing pacakges

Reply:
Adding environment diff in the revamp doc